SECURITY.md

```diff
@@ -0,0 +1,53 @@
+# Security Policy
+The Royal Navy Digital Standards Policy is to avoid leaving the ecosystem worse than we found it. Meaning we are not planning to introduce vulnerabilities into the ecosystem.
+
+The Royal Navy Digital Standards team takes security vulnerabilities in the Standards Toolkit seriously.  We appreciate your efforts to responsibly disclose your
+findings, and will make every effort to acknowledge your contributions.
```

Comment:
could this be on the same line as "disclose your"?

Reply:
yes done